index.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const { spawnSync } = require('child_process');
+const os = require('os');
+const readline = require('readline');
+const { getCurrentUnixTime, clearLine, getComand,logFileName, windowsCommand, unixLikeOSCommand, timeToRefresh } = require('./helpers');
+
+if (!fs.existsSync(logFileName)) {
+  fs.writeFileSync(logFileName, '');
+}
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+function monitorProcesses() {
+  const currentPlatform = os.platform();
+  const command = getComand(currentPlatform);
+
+  setInterval(() => {
+    const currentTime = getCurrentUnixTime();
+    const result = spawnSync(command, { shell: true, encoding: 'utf-8' });
+
+    if (result.error) {
+      console.error('Error:', result.error.message);
+      return;
+    }
+
+    const output = result.stdout.trim();
+    const logLine = `Time ${currentTime} : ${output}`;
```

Comment:
It just needs an extra space at the end, so every log is separated. Now it looks like this:

Time 1694404800 : 7.5  0.6 /System/Library/PrivateFrameworks/SkyLight.framework/Resources/_WindowServerTime_ 1694404800 : 8.2  0.6 ...

Reply:
Fixed just pass data as it is from childProcess.stdout.on